JS-Algorithms/challenges.js

```diff
@@ -96,6 +151,22 @@ Since 10! === 3628800 and you sum 3 + 6 + 2 + 8 + 8 + 0 + 0
 
 const digitSum = (n) => {
   // YOUR CODE HERE...
+
+  //We find the factorial of the number
+  let factorial = 1;
+  for (let x = n; x > 0; x--) {
+    factorial *= x;
+  }
+
+  //We add the digits of the factorial
+  let sumDigit = 0;
+  let stringFactorial = String(BigInt(factorial));
```

Comment:
this variable is never reassigned so you can change it to `const` also, a question for you. In the previous function `ownPower` you used `.toString` and now you are using  `String` I just wonder if you know when to use one or the other?

Reply:
I didn't know what the difference between toString() and String() was, but do some research and I understand better, I'll take it into account in the next exercise.
Difference between String(value) and value.toString() : http://js.dokry.com/cul-es-la-diferencia-entre-string-value-vs-value-tostring.html